src/data/critical-constants-acentric-factor.ts

```diff
@@ -303,4 +303,912 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
+  },
+
+  Butyraldehyde: {
+    name: 'Butyraldehyde',
+    formula: 'C4H8O',
+    CAS: '123-72-8',
+    molecularWeight: 72.10572,
+    criticalTemperature: 537.2,
+    criticalPressure: 4.41,
+    criticalVolume: 0.258,
+    criticalCompressibilityFactor: 0.255,
+    acentricFactor: 0.282553,
+  },
+
+  ButyricAcid: {
+    name: 'Butyric acid',
+    formula: 'C4H8O2',
+    CAS: '107-92-6',
+    molecularWeight: 88.1051,
+    criticalTemperature: 615.7,
+    criticalPressure: 4.06,
+    criticalVolume: 0.293,
+    criticalCompressibilityFactor: 0.232,
+    acentricFactor: 0.675003,
+  },
+
+  Butyronitrile: {
+    name: 'Butyronitrile',
+    formula: 'C4H7N',
+    CAS: '109-74-0',
+    molecularWeight: 69.1051,
+    criticalTemperature: 585.4,
+    criticalPressure: 3.88,
+    criticalVolume: 0.291,
+    criticalCompressibilityFactor: 0.232,
+    acentricFactor: 0.3601,
+  },
+
+  CarbonDioxide: {
+    name: 'Carbon dioxide',
+    formula: 'CO2',
+    CAS: '124-38-9',
+    molecularWeight: 44.0095,
+    criticalTemperature: 304.21,
+    criticalPressure: 7.383,
+    criticalVolume: 0.094,
+    criticalCompressibilityFactor: 0.274,
+    acentricFactor: 0.223621,
+  },
+
+  CarbonDisulfide: {
+    name: 'Carbon disulfide',
+    formula: 'CS2',
+    CAS: '75-15-0',
+    molecularWeight: 76.1407,
+    criticalTemperature: 552,
+    criticalPressure: 7.9,
+    criticalVolume: 0.16,
+    criticalCompressibilityFactor: 0.275,
+    acentricFactor: 0.110697,
+  },
+
+  CarbonMonoxide: {
+    name: 'Carbon monoxide',
+    formula: 'CO',
+    CAS: '630-08-0',
+    molecularWeight: 28.0101,
+    criticalTemperature: 132.92,
+    criticalPressure: 3.499,
+    criticalVolume: 0.0944,
+    criticalCompressibilityFactor: 0.299,
+    acentricFactor: 0.0481621,
+  },
+
+  CarbonTetrachloride: {
+    name: 'Carbon tetrachloride',
+    formula: 'CCl4',
+    CAS: '56-23-5',
+    molecularWeight: 153.8227,
+    criticalTemperature: 556.35,
+    criticalPressure: 4.56,
+    criticalVolume: 0.276,
+    criticalCompressibilityFactor: 0.272,
+    acentricFactor: 0.192552,
+  },
+
+  CarbonTetrafluoride: {
+    name: 'Carbon tetrafluoride',
+    formula: 'CF4',
+    CAS: '75-73-0',
+    molecularWeight: 88.0043,
+    criticalTemperature: 227.51,
+    criticalPressure: 3.745,
+    criticalVolume: 0.143,
+    criticalCompressibilityFactor: 0.283,
+    acentricFactor: 0.178981,
+  },
+
+  Chlorine: {
+    name: 'Chlorine',
+    formula: 'Cl2',
+    CAS: '7782-50-5',
+    molecularWeight: 70.906,
+    criticalTemperature: 417.15,
+    criticalPressure: 7.71,
+    criticalVolume: 0.124,
+    criticalCompressibilityFactor: 0.276,
+    acentricFactor: 0.0688183,
+  },
+
+  Chlorobenzene: {
+    name: 'Chlorobenzene',
+    formula: 'C6H5Cl',
+    CAS: '108-90-7',
+    molecularWeight: 112.5569,
+    criticalTemperature: 632.35,
+    criticalPressure: 4.5191,
+    criticalVolume: 0.308,
+    criticalCompressibilityFactor: 0.265,
+    acentricFactor: 0.249857,
+  },
```

Comment:
```suggestion
  },
  
```
newline missing , and after this compound as well